utils.py

```diff
@@ -352,3 +354,28 @@ def coco_evaluation(gt_filepath, prediction_filepath, image_ids, category_ids):
     cocoEval.summarize()
 
     return cocoEval.stats[1], cocoEval.stats[8]
+
+
+def dense_crf(img, output_probs, compat_gaussian=3, sxy_gaussian=1, compat_bilateral=10, sxy_bilateral=1, srgb=50):
```

Comment:
@taraspiotr I think this function should be defined in the postprocessing.py. It's far more than just an auxiliary function.